boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SPI=y
```

Comment:
One more request, 
Please also don't enable SPI because it is also not used at this time.

Reply:
> One more request, Please also don't enable SPI because it is also not used at this time.

Okay. Removed.